internal/config/mount_config.go

```diff
@@ -24,9 +24,29 @@ type LogConfig struct {
 	FilePath string      `yaml:"file-path"`
 }
 
+type FileCacheConfig struct {
+	Path                    string `yaml:"path"`
+	Size                    int64  `yaml:"size-mb"`
+	TTL                     int64  `yaml:"ttl-sec"`
+	CacheFileForRandomReads bool   `yaml:"cache-file-for-random-reads"`
+}
+
+type MetadataCacheConfig struct {
+	Capacity int64 `yaml:"capacity"`
+	TTL      int64 `yaml:"ttl-sec"`
+}
+
+type TypeCacheConfig struct {
```

Comment:
dont do these extra changes. We are going to merge typecache and stat-cache ttl in this release